### DISCUSSION TITLE
Audible reloads

### DISCUSSION THREAD

**Discussion by AdamTadeusz** [NeotokyoRebuild/neo#626] [Ideas]
According to https://github.com/NeotokyoRebuild/neo/issues/583 , in the original game only the person reloading their weapon can hear their own reload.

The current behavior on the master branch as of v8.1 is that reloads can be heard by everyone in the vicinity, except the sound sounds like its coming from inside the head of whoever is listening.

There is a PR open https://github.com/NeotokyoRebuild/neo/pull/610 to change this behavior so that reload sounds can be located in the same way as footsteps.

Is this a good change, or would it be better to stick to parity and make reload sounds quiet to other people

**Comment by brysondev** [comment]
Positional reloads would be a nice touch so you know if someone is hiding trying to reload instead of just guessing.
